crates/shrs_core/src/builtin/cd.rs

```diff
@@ -35,8 +36,8 @@ impl BuiltinCmd for CdBuiltin {
                 if let Ok(old_pwd) = rt.env.get("OLDPWD") {
                     PathBuf::from(old_pwd)
                 } else {
-                    eprintln!("no OLDPWD");
-                    return Ok(BuiltinStatus::error());
+                    ctx.out.eprintln("no OLDPWD")?;
```

Comment:
does this still have the ability to format strings? is it still a variadic function

Reply:
will make it variadic